blocks/init/src/Blocks/components/pagination/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+	"$schema": "https://raw.githubusercontent.com/infinum/eightshift-frontend-libs/develop/schemas/component.json",
+	"componentName": "pagination",
+	"title": "Pagination",
+	"componentClass": "pagination",
+	"example": {
+		"attributes": {
+			"paginationUse": true
+		}
+	},
+	"components": {
+		"icon": "icon"
+	},
+	"attributes": {
+		"paginationUse": {
```

Comment:
This is not required since it's just a component, without any editor component or options, so there's no real way to toggle it.